src/styles/blocks/stars.scss

```diff
@@ -0,0 +1,15 @@
+.stars {
+  display: flex;
+
+  &__star {
```

Comment:
Where are the margins? It is very tight between the stars

Reply:
<img width="221" alt="npm test" src="https://github.com/mate-academy/layout_catalog/assets/10390518/d1912639-4ed7-45b9-8b47-6deb4420543e">
<img width="189" alt="npm start" src="https://github.com/mate-academy/layout_catalog/assets/10390518/f0dd104f-0617-4a36-a515-f66c35b43911">